packages/authorization-process/src/routers/AuthorizationRouter.ts

```diff
@@ -665,7 +670,9 @@ const authorizationRouter = (
               filters: {
                 name,
                 userIds: userIds?.map(unsafeBrandId<UserId>),
-                producerId: unsafeBrandId(producerId),
+                producerId: producerId
+                  ? unsafeBrandId<TenantId>(producerId)
+                  : undefined,
```

Comment:
See comment above, this should be kept mandatory 
```suggestion
        producerId: unsafeBrandId(producerId),
```